platform/quality-assurance/create-an-issue.md

```diff
@@ -0,0 +1,20 @@
+# How to Create an Issue
+
+1. Navigate to relevant repo in GitHub.  Example: [vets-website](https://github.com/department-of-veterans-affairs/vets-website/issues)
+2. Click New Issue
+3. Include relevant details required to reproduce the issue
```

Comment:
can we put  all these details  into the issue template, rather than in the documentation? would prefer putting it closer to where the issue gets filled out / harder to miss

Reply:
That makes sense, and the thought definitely occurred to me.  I wasn't sure what the process is for adding a template for use in GH.  I can make the template creation a to-do for this sprint.